dts/bindings/sensor/ti,tmp11x.yaml

```diff
@@ -60,3 +60,16 @@ properties:
       GPIO pin used for alert/interrupt functionality.
       Supported on TMP116, TMP117, and TMP119 devices.
       Alert pin functionality depends on Therm/alert modus.
+  store-attr-values:
+    type: boolean
+    description: |
+      Enable storage of sensor attribute values in EEPROM. On reset, the device
+      goes through a POR sequence that loads the values programmed in the
+      EEPROM into the respective register map locations.
+
+      The following values are supported by the sensor but might not be
+      implemented by the driver:
```

Comment:
A binding should not be referring to any particular "driver" implementation (https://docs.kernel.org/devicetree/bindings/writing-bindings.html)

Reply:
Thanks! I removed the driver reference.